iml-postgres/src/lib.rs

```diff
@@ -17,6 +21,35 @@ pub use tokio_postgres::{
 };
 use tokio_postgres::{tls::NoTlsStream, Connection, NoTls, Socket};
 
+pub async fn get_db_pool() -> Result<PgPool, sqlx::Error> {
+    let mut opts = PgConnectOptions::default().username(&iml_manager_env::get_db_user());
+
+    opts = if let Some(x) = iml_manager_env::get_db_host() {
```

Comment:
Would it be cleaner to do something like:
```
opts = iml_manager_env::get_db_host().map_or(opts, |x| opts.host(&x));
opts = iml_manager_env::get_db_name().map_or(opts, |x| opts.database(&x));
opts = iml_manager_env::get_db_password().map_or(opts, |x| opts.password(&x));
```
?

Reply:
Subjective I suppose